K2Bridge.Tests.UnitTests/Models/Response/DateHistogramBucketTests.cs

```diff
@@ -17,11 +17,11 @@ public void Create_WithDateTime_ReturnsISOString()
         {
             // Arrange
             DataTable table = new DataTable();
-            table.Columns.Add("Timestamp", typeof(DateTime)).DateTimeMode = DataSetDateTime.Utc;
-            table.Columns.Add("Count", typeof(int));
+            table.Columns.Add("timestamp", typeof(DateTime)).DateTimeMode = DataSetDateTime.Utc;
```

Comment:
why did we have to change these column names?

Reply:
It was not really needed, I updated these columns because names were in lowercase in other tests - and adx returns the columns in lowercase too. Anyway, tests were working fine before, and still works after.